docs/source/examples/supplychain/overview.rst

```diff
@@ -0,0 +1,100 @@
+*******************
+SupplyChain Example
+*******************
+
+In this example provided the infrastructure necessary track physical objects
+thru a real world supply chain on the Sawtooth Ledger. Obviously we can not put
```

Comment:
@ojalatodd I expect other examples to be added to the docs/source/examples.rst document over time.  Such as the "private utxo" documents I am working on now In the future xo, intkey, etc can all have dedicated section where their specs and other parts of the implementation are described.